### ISSUE TITLE
Automatically bind metrics for new CircuitBreaker from CircuitBreakerRegistry

### ISSUE THREAD

**Issue by efenderbosch** [resilience4j/resilience4j#450]
We are creating a custom ``CircuitBreakerConfig`` and ``CircuitBreakerRegistry``. When a new ``CircuitBreaker`` is created from the registry, metrics are not bound. If the ``CircuitBreakerConfig`` had a ``bindMetrics`` attribute the ``CircuitBreakerRegistry`` could automatically bind metrics before returning the new ``CircuitBreaker``.

This wasn't a big issue before because I could use code like
```java
CircuitBreaker circuitBreaker = circuitBreakerRegistry.circuitBreaker(name);
CircuitBreakerMetrics.ofIterable(Collections.singletonList(circuitBreaker)).bindTo(meterRegistry);
```

However, a recent PR is removing ``CircuitBreakerMetrics`` and only ``TaggedCircuitBreakerMetrics`` remains. ``TaggedCircuitBreakerMetrics::bindTo(CircuitBreakerRegistry)`` loops through all  ``CircuitBreaker`` instances in the ``CircuitBreakerRegistry`` and rebinds them even if they already have metrics bound. Either ``CircuitBreakerRegistry`` should automatically bind metics for newly created ``CircuitBreaker`` instances or ``TaggedCircuitBreakerMetrics`` should have a method to bind a single ``CircuitBreaker`` to the ``MeterRegistry``. My preference would be for the registry to automatically bind them based on a config setting.

This is probably also relevant for retries, bulkheads, etc.

**Comment by RobWin** [comment]
Hi,

thank you for using Resilience4j.
This is fixed in the upcoming release v0.15.0 for CircuitBreaker, Retry, Bulkhead and RateLimiter.

Please see:

https://github.com/resilience4j/resilience4j/blob/master/resilience4j-micrometer/src/main/java/io/github/resilience4j/micrometer/tagged/TaggedCircuitBreakerMetrics.java#L140-L148

**Comment by RobWin** [comment]
Feel free to close the issue, if it solves your problem.
